pkg/tfvars/alibabacloud/bootstrap_ignition.go

```diff
@@ -0,0 +1,76 @@
+package alibabacloud
+
+import (
+	"bytes"
+	"encoding/pem"
+	"fmt"
+
+	ignutil "github.com/coreos/ignition/v2/config/util"
+	igntypes "github.com/coreos/ignition/v2/config/v3_2/types"
+	"github.com/vincent-petithory/dataurl"
+
+	"github.com/openshift/installer/pkg/asset/ignition"
+)
+
+// Ref: pkg/tfvars/aws/bootstrap_ignition.go
```

Comment:
Okey, I may modify some aws part of the code, is this allowed？

Reply:
Yes. Preferably we would have a separate commit that includes just the changes extracting this code from the AWS platform, making it clear that it is being done so that it can be used by other platforms.